code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/TokenResourceManager.java

```diff
@@ -24,11 +32,12 @@ protected Object createInternal(String type, ApiRequest request) {
         if (!StringUtils.equals(TOKEN, request.getType())) {
             return null;
         }
-        try {
-            return githubTokenHandler.getGithubAccessToken(request);
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+        return githubTokenHandler.getGithubAccessToken(request);
+    }
+
+    @Override
+    protected Object listInternal(SchemaFactory schemaFactory, String type, Map<Object, Object> criteria, ListOptions options) {
```

Comment:
Again, `listInternal` should return a list.  Let's talk to @vincent99 about this.